hatchet_sdk/clients/admin.py

```diff
@@ -262,30 +257,22 @@ async def run_workflow(
     async def run_workflows(
         self,
         workflows: list[WorkflowRunDict],
-        options: TriggerWorkflowOptions | None = None,
-    ) -> List[WorkflowRunRef]:
+        options: TriggerWorkflowOptions = TriggerWorkflowOptions(),
+    ) -> list[WorkflowRunRef]:
         if len(workflows) == 0:
             raise ValueError("No workflows to run")
         try:
             if not self.pooled_workflow_listener:
                 self.pooled_workflow_listener = PooledWorkflowRunListener(self.config)
 
-            namespace = self.namespace
-
-            if (
-                options is not None
-                and "namespace" in options
-                and options["namespace"] is not None
-            ):
-                namespace = options["namespace"]
-                del options["namespace"]
```

Comment:
@grutt is there a reason we were deleting this key before? I can do this too if it's important, but just wanted to check